src/home-page/index.js

```diff
@@ -192,7 +208,8 @@ const HomePage = () => {
 
 
             <div className="row m-5 background-teal">
-                <a href="http://localhost:3000/map" className="col-8 fs-2 fw-bold text-white d-flex align-items-center justify-content-center">
+                <a href="http://localhost:3000/map"
```

Comment:
Eventually, our site won't be hosted on `localhost` so we want to keep this link relative to whatever the base URL is

```suggestion
                <a href="/map"
```

Reply:
Also, note that we have multiple links to `/map` throughout the application. To make the lives of developers easier, we can create an dictionary that allows us to quickly reference the various links we have available. Ex:

```ts
export const Routes = {
  MAP: '/map'
}
```

then here we can just do

```tsx
<a href={Routes.MAP} />
```